charts/pega/README.md

```diff
@@ -429,12 +429,30 @@ tier:
 
 ### Pega configuration files
 
-While default configuration files are included by default, the Helm charts provide extension points to override them with additional customizations.  To change the configuration file, specify a relative path to a local implementation to be injected into a ConfigMap.
+While default configuration files are included by default, the Helm charts provide extension points to override them with additional customizations.  To change the configuration file, specify the replacement implementation to be injected into a ConfigMap.
```

Comment:
While Pega includes default configuration files in the Helm charts, the charts provide extension points to override the defaults with additional customizations.  To change the configuration file, specify the replacement implementation to be injected into a ConfigMap.